### DISCUSSION TITLE
Option to automatically whitelist video creator

### DISCUSSION THREAD

**Discussion by Firecul** [ThioJoe/YT-Spammer-Purge#399] [Ideas]
In some smaller channels I've scanned videos for, the video creator can tend to get flagged for similar comments while saying "thanks for watching" for example to people commenting. Having the option in the results or in the config to exclude a video's creator should cut those false positives out.

**Comment by ThioJoe** [comment]
Oh that's not supposed to happen, that's a bug. Creator comments should always been ignored, I forgot to do it for the duplicate scanning.

**Comment by ThioJoe** [comment]
Ok should be fixed with 2.12.1

**Comment by Firecul** [comment]
I didn't even think that it was a bug. I figured you'd be fed up with me submitting issues too 😅

**Comment by ThioJoe** [comment]
nah I'd rather know about every single bug haha
